examples/diagram-fit/index.html

```diff
@@ -0,0 +1,115 @@
+                <h2>Fit on load</h2>
+                <p>If the diagram is too large, it does not fit within the viewport. <br>
+                    Use the radio buttons to choose how fit the diagram in the viewport.
+                </p>
+
+                <div class="columns" style="margin-top: 2.5rem">
+                    <div id="controls" class="column col-2 p-centered">
+                        <div class="m-2 p-2 border">
+                            <h5>Fit type</h5>
+                            <div class="radio">
+                                <label><input type="radio" name="fitType" value="None" checked> None</label>
+                            </div>
+                            <div class="radio">
+                                <label><input type="radio" name="fitType" value="HorizontalVertical"> Horizontal-Vertical</label>
+                            </div>
+                            <div class="radio">
+                                <label><input type="radio" name="fitType" value="Horizontal"> Horizontal</label>
+                            </div>
+                            <div class="radio">
+                                <label><input type="radio" name="fitType" value="Vertical"> Vertical</label>
+                            </div>
+                            <div class="radio">
+                                <label><input type="radio" name="fitType" value="Center"> Center</label>
+                            </div>
+                        </div>
+
+                        <div class="" style="margin-top: 1.5rem">
+                            <div class="dropdown">
+                                <a id="toggle-dropdown-diagram-choice" href="#" class="btn btn-primary dropdown-toggle" tabindex="0" style="background: black">
+                                    Select a BPMN Diagram <span class="icon icon-caret icon-in-the-middle"></span>
+                                </a>
+                                <ul class="menu" id="dropdown-diagram-choice">
+                                    <li><a href="#" class="btn btn-link">horizontal diagram</a></li>
+                                    <li><a href="#" class="btn btn-link">vertical diagram</a></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div id="bpmn-viewport" class="column col-10 col-mx-auto bpmn-viewport"/>
+                </div>
+            </div>
+        </section>
+
+        <!-- load bpmn-visualization -->
+        <!-- To uncomment when the release of the 0.5.1 is done -->
+        <!--    <script src="https://unpkg.com/bpmn-visualization@0.5.1/dist/bpmn-visualization.js"></script>-->
+        <!-- To delete when the release of the 0.5.1 is done -->
+        <script src="../../demo/0.5.1-alpha/bpmn-visualization.js"></script>
+        <script src="./diagram-fit.js" type="module"></script>
+    </body>
+</html>
```

Comment:
:warning: there is a mix here!
the page load the IIFE bundle but the js script used the ESM internal dev module.
It would be more convenient to only use the IIFE bundle:
- the page can be load without webserver
- avoid dependencies to the ESM internal dev module not available publicly

This is something noticed in https://github.com/process-analytics/bpmn-visualization-js/issues/599

In addition, we probably are going to release a 0.6.0 version, so I would avoid specifying any version here

Reply:
solved